src/auxiliary/p4est.jl

```diff
@@ -84,29 +84,29 @@ end
 # Save `p4est` data to file
 # 2D
 function save_p4est!(file, p4est::PointerOrWrapper{p4est_t})
-    # Don't save user data of the quads
-    p4est_save(file, p4est, false)
+    # Also save user data to be able to reload it for restarts
+    p4est_save(file, p4est, true)
 end
 
 # 3D
 function save_p4est!(file, p8est::PointerOrWrapper{p8est_t})
-    # Don't save user data of the quads
-    p8est_save(file, p8est, false)
+    # Also save user data to be able to reload it for restarts
+    p8est_save(file, p8est, true)
 end
 
 # Load `p4est` from file
 # 2D
 function load_p4est(file, ::Val{2})
     conn_vec = Vector{Ptr{p4est_connectivity_t}}(undef, 1)
     comm = P4est.uses_mpi() ? mpi_comm() : C_NULL # Use Trixi.jl's MPI communicator if p4est supports MPI
-    p4est_load_ext(file, comm, 0, 0, 1, 0, C_NULL, pointer(conn_vec))
+    p4est_load_ext(file, comm, 2 * sizeof(Int), 1, 1, 0, C_NULL, pointer(conn_vec))
```

Comment:
I know there wasn't a comment here before, but maybe you can add one that explains what this line does?